extensions/amp-analytics/0.1/activity-impl.js

```diff
@@ -119,6 +123,15 @@ const ACTIVE_EVENT_TYPES = [
   'mousedown', 'mouseup', 'mousemove', 'keydown', 'keyup',
 ];
 
+/**
+ * @param {!../../../src/service/ampdoc-impl.AmpDoc} ampDoc
+ * @return {Activity}
+ */
+export function activityServiceForTesting(ampDoc) {
```

Comment:
Done. Thanks for catching this. I've renamed the method to install ActivityServiceForTesting since it's still exclusively used for testing.